zipline/finance/risk/cumulative.py

```diff
@@ -281,6 +286,12 @@ def update(self, dt, algorithm_returns, benchmark_returns, leverage):
         self.max_leverage = self.calculate_max_leverage()
         self.max_leverages[dt_loc] = self.max_leverage
 
+        self.cvar[dt_loc] = conditional_value_at_risk(
+            returns=pd.DataFrame(self.algorithm_returns[-CVAR_LOOKBACK_DAYS:]),
```

Comment:
That's a great point and something I realized right after putting this up; what I'm unsure of is how to handle the `weights` parameter in the case that we pass an array (or anything of ndim 1). In the case of an array, do we just ignore the weights parameter? Seems odd if it's required but does nothing. (either way you're right that I should change this not to construct a DataFrame).